scripts/core/editor3/store/index.js

```diff
@@ -79,14 +86,20 @@ function onChange(content) {
  * use available HTML. If none are available, an empty ContentState is created.
  */
 function getInitialContent(props) {
-    // we have an editor state stored in the DB
-    if (props.editorState && typeof props.editorState === 'object') {
-        return applyInlineStyles(convertFromRaw(props.editorState)).contentState;
+    // we have an editor state stored in the DB for body field
+    if (props.field === 'body_html' && props.item.editor_state &&
```

Comment:
why do it here and make editor do some field specific behavior?

Reply:
The editor_state is used only for body_html. We can add a new parameter to specify if the editor_state is used or not.

Reply:
ok so we could use the old editorState param and only set it for body_html right?

Reply:
I will use a boolean parameter for this.